R/epi_df.R

```diff
@@ -111,6 +111,51 @@ NULL
 #' @return An `epi_df` object.
 #'
 #' @export
+#' @examples 
+#' # Convert a `tsibble` that has county code as an extra key
+#' ex1 <- tsibble::tibble(
+#'   geo_value = rep(c("ca", "fl", "pa"), each = 3),
+#'   county_code = c(06059,06061,06067,
+#'                   12111,12113,12117,
+#'                   42101, 42103,42105),
```

Comment:
County codes should likely be strings, in order to preserve any leading zeros without extra formatting.  E.g., that would help prevent issues conforming with [the COVID-19 Forecat Hub format](https://github.com/reichlab/covid19-forecast-hub/blob/master/data-processed/README.md#location).